### ISSUE TITLE
International ID V2 returns API error that wrong mime format is used when creating from Base64 string.

### ISSUE THREAD

**Issue by psihius** [mindee/mindee-api-php#57]
### Prerequisites

Put an X between the brackets on this line if you have done all of the following:

* [X] Reproduced the problem or exposed a new need
* [X] Checked the GitHub existing issues

### Description

When using `sourceFromBase64String`, the CURL file object for some reason here https://github.com/mindee/mindee-api-php/blob/effc7b1f25e8193e1a66c46d47e0b8d16b6283ef/src/Http/Endpoint.php#L121 ignores the mime type and ends up submitting data as `application/octet-stream` 

### Steps to Reproduce

1. Grab a base64 of a valid file.
2. Submit it to the API like this
```
$inputSource = $mindeeClient->sourceFromB64String($fileData, 'file.jpeg');
$response = $mindeeClient->enqueueAndParse(InternationalIdV2::class, $inputSource);
```

**Expected behavior:**

Returns a document inference object

**Actual behavior:**

PRoduces API error stating wrong file mime type.

**Reproduces how often:**

100%

### Versions

mindee/mindee 1.5.0
php-fpm 8.2 official docker image (debian 12)

### Additional Information

IF we change the line https://github.com/mindee/mindee-api-php/blob/effc7b1f25e8193e1a66c46d47e0b8d16b6283ef/src/Http/Endpoint.php#L121 to 
```
$postFields = ['document' => $fileCurl->readContents()[1]);
```
Then it works as expected, so either cURL is doing something wrong with that Base64 encoded data or the server is wrongly detecting the contents of the submitted data. I did debug it and mime type on the curl file object itself is correct - `image/jpg`

**Comment by sebastianMindee** [comment]
Seems like I forgot to decode the string when adding it to the file... Thank you, will be fixed shortly!

**Comment by psihius** [comment]
> Seems like I forgot to decode the string when adding it to the file... Thank you, will be fixed shortly!

Thank you. I will update as soon as it's merged :)

**Comment by sebastianMindee** [comment]
It's fixed in #60, but we'll release later today with some more additions.
